src/Rings/mpoly.jl

```diff
@@ -742,15 +742,20 @@ function jacobi_ideal(f::MPolyRingElem)
 end
 
 @doc Markdown.doc"""
-    jacobi_matrix(g::Vector{<:MPolyRingElem})
+    jacobi_matrix([R::MPolyRing,] g::Vector{<:MPolyRingElem})
 
-Given an array ``g=[f_1,...,f_m]`` of polynomials over the same base ring,
+Given an array ``g=[f_1,...,f_m]`` of polynomials over the same base ring `R`,
 this function returns the Jacobian matrix ``J=(\partial_{x_i}f_j)_{i,j}`` of ``g``.
 """
 function jacobi_matrix(g::Vector{<:MPolyRingElem})
```

Comment:
The signature of the actual method did not change.

Reply:
This should be the correct signature for optional arguments.